src/main/kotlin/org/misarch/catalog/event/CatalogEvents.kt

```diff
@@ -19,6 +19,11 @@ object CatalogEvents {
      */
     const val PRODUCT_VARIANT_VERSION_CREATED = "catalog/product-variant-version/created"
 
+    /**
+     * Topic for category creation events
+     */
+    const val CATEGORY_CREATED = "catalog/category/created"
```

Comment:
`CATEGORY_UPDATED` is postponed for another PR?

Reply:
currently categories are completely immutable

Reply:
Fine for me